smartcontractkit/external-adapters-js
# change rpc url to lcd url in solana view function EA

## Closes 29911

## Description

 - Change name of `RPC_URL` env var to `LCD_URL` in the Solana View Function EA
 
## Changes

- Change the name of `RPC_URL` env var to `LCD_URL` in the Solana View Function EA
- Update ReadMe

## Steps to Test

1. Run EA by setting `LCD_URL` env var
2. Verify that you get a successful response from the EA

## Quality Assurance
